subsys/bluetooth/host/hci_core.c

```diff
@@ -1265,6 +1265,12 @@ static void enh_conn_complete(struct bt_hci_evt_le_enh_conn_complete *evt)
 	if (IS_ENABLED(CONFIG_BT_CENTRAL) &&
 	    IS_ENABLED(CONFIG_BT_WHITELIST) &&
 	    evt->role == BT_HCI_ROLE_MASTER) {
+		/*
+		 * Clear auto conn even if we are not able to add connection
```

Comment:
Minor: please start the text on the first comment line, i.e. `/* Clean auto...`

I can see we haven't been completely consistent with this, which is probably reflected in the inconsistency in your own patches (here you do this, other comments you don't), however I'd prefer if we'd standardize on the starting with the first line. It's more compact, and (at least for me) it doesn't really hurt the readability.

Reply:
I copy pasted the comment and edited it. Updated both places.